db_stress_tool/cf_consistency_stress.cc

```diff
@@ -121,23 +121,48 @@ class CfConsistencyStressTest : public StressTest {
     Status s;
     bool is_consistent = true;
 
+    ReadOptions readoptionscopy = readoptions;
+    readoptionscopy.fill_cache = thread->rand.OneIn(2);
```

Comment:
should we have a function that randomizes `ReadOptions` in such a way that the expected results are unchanged? It feels there are a lot of combinations, especially for iterators.

Reply:
Also would it be easier to call `TestGet()` multiple times?